aarch64_linux/aarch64_ci_setup.sh

```diff
@@ -19,7 +19,16 @@ curl -L -o /mambaforge.sh https://github.com/conda-forge/miniforge/releases/late
 chmod +x /mambaforge.sh
 /mambaforge.sh -b -p /opt/conda
 rm /mambaforge.sh
-/opt/conda/bin/conda config --set ssl_verify False
-/opt/conda/bin/conda install -y -c conda-forge python=${DESIRED_PYTHON} numpy pyyaml setuptools patchelf pygit2 openblas ninja scons
+source /opt/conda/etc/profile.d/conda.sh
+conda config --set ssl_verify False
+conda create -y -c conda-forge -n aarch64_env python=${DESIRED_PYTHON}
+conda activate aarch64_env
+conda install -y -c conda-forge numpy
+conda install -y -c conda-forge pyyaml
+conda install -y -c conda-forge patchelf
+conda install -y -c conda-forge pygit2
+conda install -y -c conda-forge openblas
+conda install -y -c conda-forge ninja
+conda install -y -c conda-forge scons
```

Comment:
do you mind merging it into one command and pinning versions here?
```suggestion
conda install -y -c conda-forge numpy==1.26 pyyaml patchelf pygit2 openblas ninja scons
```

Reply:
I did that because it makes it a lot simpler to understand which package conda cannot find and leads to 200lines of "un-resolvable unvironment". happy to change it back if you prefer.

Reply:
Updated and added all the pins based on the last run from the core CI